EIPS/contract_event_subscription.md

```diff
@@ -0,0 +1,818 @@
+---
+eip: draft
+title: Contract Event Subscription
+description: Allows contracts to subscribe to and react to events emitted by other contracts with gas-bounded execution
+author: Lucas Cullen (@bitcoinbrisbane) <lucas@bitcoinbrisbane.com.au>
+discussions-to: https://ethereum-magicians.org/t/xxxxx
```

Comment:
Please create a discussions topic using the template https://ethereum-magicians.org/c/eips/5